pandas/tests/indexes/period/test_indexing.py

```diff
@@ -523,7 +523,7 @@ def test_get_indexer2(self):
         tol_bad = [
             Timedelta("2 hour").to_timedelta64(),
             Timedelta("1 hour").to_timedelta64(),
-            np.timedelta64(1, "M"),
+            np.timedelta64(1, "ME"),
```

Comment:
this is `numpy`'s `timedelta64`, which we're not changing anything in, so let's keep this one as it is

Reply:
thanks, I corrected my mistake.